# TEST-TASK___________________________________
def test_2(temp):
    try:
        assert len(temp) == 1 and \
               "В первом варианте объект range в операторе in каждый раз возвращает новый итератор и перебор последовательности начинается с начала. Во втором варианте явно указывается итератор, который в строчке res2 = 2022 in r продолжает перебор последовательности с числа 2023." in temp
    except:
        print("Пока неправильно попробуйте другие варианты ...")
    else:
        [print(_) for _ in temp]
        print()
        print("Правильно .")
